src/mock/service-mock.js

```diff
@@ -0,0 +1,62 @@
+import {WAYPOINT_TYPE, DESTINATION_COUNT, OFFER_COUNT, POINT_COUNT} from '../constant.js';
+import {getRandomArrayElement, getRandomInteger} from '../utils.js';
+import {generateDestination} from './destination-mock.js';
+import {generatePoint} from './point-mock.js';
+import {generateOffer} from './offer-mock.js';
+
+
+export default class MockService {
+  destinations = [];
+  offers = [];
+  points = [];
+
+  constructor() {
+    this.destinations = this.generateDestinations();
+    this.offers = this.generateOffers();
+    this.points = this.generatePoints();
+  }
+
+  getDestinations() {
+    return this.destinations;
+  }
+
+  getOffers() {
+    return this.offers;
+  }
+
+  getPoints() {
+    return this.points;
+  }
+
+  generateDestinations() {
+    return Array.from(
+      {length: DESTINATION_COUNT},
+      () => generateDestination()
```

Comment:
Если не надо прокидывать аргументы то можно более коротко
```suggestion
      ({length: DESTINATION_COUNT}, generateDestination)
```